### ISSUE TITLE
Is this project OS X only?

### ISSUE THREAD

**Issue by dolence** [nseaSeb/Sysex77#2]

**Comment by nseaSeb** [comment]
No, it's not OS X only, but I don't have the means to compile it for other environments.
Juce allows compilation for Mac, Linux, and Windows.
